docs/rest_api.md

```diff
@@ -1200,7 +1200,7 @@ Authorization requirements:
 
 #### user_set_admin
 
-'PUT /auth/basic/user/<user>/set_admin'
+'PATCH /auth/basic/user/<user>/is_admin'
```

Comment:
With the PATCH verb it makes more sense to just do `/auth/basic/user/<user>`; the thing you're patching is the user.